### ISSUE TITLE
[Bug]: Shuffle without warping should work, yet [Warning] ControlNet finds unexpected mis-alignment in tensor shape.

### ISSUE THREAD

**Issue by torridgristle** [Mikubill/sd-webui-controlnet#1575]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

Shuffle used to work without warning if used as intended with the shuffle pre-processor that warps the image. Currently it's giving the warning no matter what, but it wasn't previously the case so this is an issue from before whenever this changed.

However, if I disable the preprocessor by setting it to None, I'll get `[Warning] ControlNet finds unexpected mis-alignment in tensor shape.` in the console 13 times each step. It does seem to still work, as right after printing this warning it seems to resize the tensor via nearest neighbor anyway.

I thought that perhaps I was missing some kind of resize action, so I modified the `shuffle()` function to skip the `ContentShuffleDetector()` and just return the img after it was resized and remove_pad. Best I can tell, it should still be a 512x512 image either way if I'm generating a 512x512 image with a 512x512 reference image for shuffle. I don't know why this is happening.

It should work without shuffling according to https://github.com/lllyasviel/ControlNet-v1-1-nightly#controlnet-11-shuffle
> In fact, since the ControlNet is trained to recompose images, we do not even need to shuffle the input - sometimes we can just use the original image as input.

### Steps to reproduce the problem

1. Disable shuffle preprocessor
2. Run
3. Get "[Warning] ControlNet finds unexpected mis-alignment in tensor shape." 13 times each step.

### What should have happened?

It should have not had a mis-alignment at all,

### Commit where the problem happens

webui: 1.3.2
https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/baf6946e06249c5af9851c60171692c44ef633e0
controlnet: 1.1.218
https://github.com/Mikubill/sd-webui-controlnet/commit/1b2aa4a96e9241b96abfbce249284ed17fc98b11

### What browsers do you use to access the UI ?

Mozilla Firefox

### Command Line Arguments

```Shell
--gradio-img2img-tool color-sketch --opt-sdp-no-mem-attention --opt-channelslast --enable-insecure-extension-access --gradio-queue --no-gradio-queue --lowram

Just realized I had both queue options still there, I'll need to fix that.
```

### List of enabled extensions

ControlNet, Lora block weight, embedding inspector

### Console logs

```Shell
Startup time: 57.7s (import torch: 8.6s, import gradio: 1.4s, import ldm: 1.0s, other imports: 2.4s, setup codeformer: 0.1s, list SD models: 6.8s, load scripts: 32.5s, reload hypernetworks: 0.3s, create ui: 4.3s, gradio launch: 0.2s).
Creating model from config: /content/stable-albedo/configs/v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: /content/stable-albedo/models/VAE/vae-ft-mse-840000-ema-pruned.ckpt
Applying optimization: Doggettx... done.
Textual inversion embeddings loaded(550): (...)
Model loaded in 84.6s (load weights from disk: 43.9s, create model: 1.3s, apply weights to model: 3.3s, apply channels_last: 1.8s, apply half(): 1.2s, load VAE: 1.6s, move model to device: 0.6s, load textual inversion embeddings: 30.9s).
2023-06-06 20:04:26,888 - ControlNet - INFO - Loading model: control_v11e_sd15_shuffle [526bfdae]
2023-06-06 20:04:33,939 - ControlNet - INFO - Loaded state_dict from [/content/stable-albedo/extensions/sd-webui-controlnet/models/control_v11e_sd15_shuffle.pth]
2023-06-06 20:04:33,939 - ControlNet - INFO - Loading config: /content/stable-albedo/extensions/sd-webui-controlnet/models/control_v11e_sd15_shuffle.yaml
2023-06-06 20:04:39,464 - ControlNet - INFO - ControlNet model control_v11e_sd15_shuffle [526bfdae] loaded.
2023-06-06 20:04:39,520 - ControlNet - INFO - Loading preprocessor: none
2023-06-06 20:04:39,520 - ControlNet - INFO - preprocessor resolution = 64
  0% 0/20 [00:00<?, ?it/s][Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.

  5% 1/20 [00:03<01:13,  3.84s/it][Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
[Warning] ControlNet finds unexpected mis-alignment in tensor shape.
```

### Additional information

Is it an issue with global average pooling? I think shuffle uses this, but I don't know why it wouldn't take effect just because the image warping was skipped before being sent to the shuffle model. I'm stumped.

**Comment by lllyasviel** [comment]
reproduced. fill fix later.

**Comment by lllyasviel** [comment]
fixed in 1.1.219
feel free to reopen is not fixed
